setup.py

```diff
@@ -45,5 +45,11 @@
     dependency_links=[
         "https://github.com/google-research/bleurt/archive/b610120347ef22b494b6d69b4316e303f5932516.zip#egg=bleurt",
     ],
-    extras_require={"dev": ["pytest", "black", "pre-commit"]},
+    extras_require={
```

Comment:
Not really happy with the dev version installation but this will be changed once wandb v0.13.6 becomes GA